IdnoPlugins/IndiePub/Pages/MicroPub/Endpoint.php

```diff
@@ -104,6 +104,7 @@ function post()
                     }
                       if ($type == 'checkin')  {
```

Comment:
is this working for you? we're inside the check that `$type == 'entry'`, so I would expect it to be impossible for it to be 'checkin' here